common/src/api/external/mod.rs

```diff
@@ -1446,14 +1457,9 @@ pub struct Disk {
     #[serde(flatten)]
     pub identity: IdentityMetadata,
     pub project_id: Uuid,
-    /// ID of snapshot from which disk was created, if any
-    pub snapshot_id: Option<Uuid>,
-    /// ID of image from which disk was created, if any
-    pub image_id: Option<Uuid>,
     pub size: ByteCount,
     pub block_size: ByteCount,
     pub state: DiskState,
-    pub device_path: String,
     pub disk_type: DiskType,
```

Comment:
perhaps disk_details or disk_info

Reply:
Kind of generic but it's probably better than `disk_type` since it's not just a type. I like `disk_details` better than `disk_info`. What about `disk_type_details` to convey that they are details specific to the type?

Reply:
I think "details" implies "specific to whatever the fuck this thing is", but I'm happy to let you have this one since I had the last one ;-)